components/Tabs/Tabs.js

```diff
@@ -1,62 +1,55 @@
-
 class TabLink {
   constructor(element) {
     // Assign this.element to the passed in DOM element
-    // this.element;
-    
+    this.element = element;
     // Get the custom data attribute on the Link
-    // this.data;
-    
+    this.data = this.element.dataset.tab;
     // Using the custom data attribute get the associated Item element
-    // this.itemElement;
-    
+    this.itemElement = document.querySelector(`.tabs-item[data-tab="${this.data}"]`);
     // Using the Item element, create a new instance of the TabItem class
-    // this.tabItem;
-    
+    this.tabItem = new TabItem(this.itemElement);
     // Add a click event listener on this instance, calling the select method on click
-
-  };
+    this.element.addEventListener("click", () => {
+      this.select();
+    });
+  }
 
   select() {
     // Get all of the elements with the tabs-link class
-    // const links;
-
+    const links = document.querySelectorAll(".tabs-link");
     // Using a loop or the forEach method remove the 'tabs-link-selected' class from all of the links
-    // Array.from(links).forEach();
-
+    Array.from(links).forEach(link => {
```

Comment:
You don't need to do Array.from because `foreach` is a method on nodelist as well.